src/gui/addnewtorrentdialog.ui

```diff
@@ -6,332 +6,979 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>414</width>
-    <height>630</height>
+    <width>936</width>
+    <height>500</height>
    </rect>
   </property>
+  <property name="sizePolicy">
+   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
+    <horstretch>0</horstretch>
+    <verstretch>0</verstretch>
+   </sizepolicy>
+  </property>
+  <property name="minimumSize">
+   <size>
+    <width>936</width>
+    <height>500</height>
```

Comment:
Found it why Splitter won't resize, for some reason these are not 0.

Reply:
Splitter resized
![2018-09-25 19-49-46](https://user-images.githubusercontent.com/2264893/46015362-4f540c80-c0fc-11e8-8f01-200d87d2b076.png)

Reply:
AFAIK the minimum size is (should be?) determined by minimumSizeHint. Also Chocobo1 said to make all values default and 0 was the default for these.

Reply:
QLayout will never resize a widget to a size smaller than the minimum size hint unless minimumSize() is set or the size policy is set to QSizePolicy::Ignore. If minimumSize() is set, the minimum size hint will be ignored.

but minimumSizeHint() not uses in our code?

Reply:
It's used by default i think unless explicitly overridden http://doc.qt.io/qt-5/qwidget.html#minimumSizeHint-prop
A try will convince you that there is some minimum size (probably this) even if they are set to 0.